src/main/basecamp.ts

```diff
@@ -22,9 +22,30 @@ const HTTP_POST_METHOD = 'post';
 const HTTP_PUT_METHOD = 'put'
 const HTTP_GET_METHOD = 'get'
 
-type HTTPResponse = GoogleAppsScript.URL_Fetch.HTTPResponse;
+const BASECAMP_API_URL = 'https://3.basecampapi.com';
+const A2N_BASECAMP_ORG_ID = '4474129';
```

Comment:
For the long term will this remain hardcoded or will this be moved to be with the rest of our config values?

Reply:
good callout. for the medium term and possibly long term this could be hardcoded since the organization will always be A2N. but it could live in config values too, though it'd always stay constant. when we flesh out our environment specific configs more, we can see what makes the most sense